src/internal/evaluation/EvaluationStorage.ts

```diff
@@ -5,13 +5,39 @@ export interface EvaluationEntity {
   userId: string
   currentEvaluationsId: string | null
   evaluations: Record<string /* featureId */, Evaluation>
+  currentFeatureTag: string | null
+  evaluatedAt: number | null
+  userAttributesUpdated: boolean
```

Comment:
@cre8ivejp Do you want to persist `userAttributesUpdated`?

I mean, when the SDK fails to notify the backend that `userAttributesUpdated: true` and a user close the app, do you want to send a request with `userAttributesUpdated: true` upon the next launch?